config/application.rb

```diff
@@ -276,7 +276,7 @@ class Application < Rails::Application
     end
 
     config.after_initialize do
-      init_dynflow
+      init_dynflow unless Foreman.in_rake?('db:drop')
```

Comment:
Are there any rake tasks where it does need to be initialized?

Reply:
For a reason I don't recall now we load the Dynflow executor in `db:seed` and `db:migrate` (see[1]). Another place where we might be using it are foreman-tasks tests.

[1] - https://github.com/Dynflow/dynflow/blob/master/lib/dynflow/rails/configuration.rb#L40

Reply:
cc @iNecas

Reply:
The reason is that at the gtime of db:seed and db:migrate, the external executor might not be running, while we would need it for some seed of migration operations.

Reply:
does this prevent us to use `db:purge`?

Reply:
as per our discussion with @adamruzicka and some testing it does. However after some investigation I found out I won't be able to use `db:purge` anyway.